devcontainers-cli/main.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = ">= 0.17"
```

Comment:
@matifali Not sure what version I should use here - do you have an idea ?

Reply:
This is fine. 0.17 is when we introduced `coder_script`